index.js

```diff
@@ -1,3 +1 @@
-function subtract(a, b) {
-  return a + b;
-}
\ No newline at end of file
+const subtract = (a, b) =>  a - b;
```

Comment:
Podemos combinar de sempre utilizar function?
```suggestion
function subtract(a, b) {
  return a - b;
}
```